contents/handbook/product/user-feedback.md

```diff
@@ -22,6 +22,26 @@ Ways to invite users for an interview:
    - PMs use cal.com for scheduling interviews, but you can choose a different tool as well.
 - If a customer has requested a feature through Slack, then message them directly.
 - Email customers who have subscribed to the feature on the [roadmap](/roadmap).
+- Email a slice of users, e.g. top users of your product, or conversely, people who churned.
+
+#### Email writing inspiration
+
+When crafting user outreach, just put yourself in the shoes of the person about to receive the message. How can you help each other by getting on that quick call?
+
+Here's an example of an email from a real project, crafted by <TeamMember name="Michael Matloka" /> to learn about the problems of top users of the Max AI beta:
+
+> Subject: Quick chat about your PostHog Max AI experience?  
+>  
+> Hey $FIRST_NAME, Michael from PostHog engineering here!  
+>  
+> I'm focused on improving PostHog's AI features, and I saw you've been using our AI assistant Max.  
+> I want to make Max 10x better for you – and it'll be a gamechanger to hear about your personal experience with it.  
+> What do you say about a 30min chat about your product-building workflow, this week or in a couple of weeks? I promise in return you'll get a better tool for your job, plus $40 of PostHog merch. :)  
+>  
+> Feel free to pick any time that suits you in my calendar at https://cal.com/posthog-michael/30min, or send me your own calendar! I'm excited to hear from you.  
```

Comment:
I am 80% sure someone will copy this and put your calendar link in lol

Reply:
I'm _always_ happy to talk to users

Reply:
Actually someone will 80% do that, so I updated to say below: _don't_ do that

Reply:
Yeah maybe remove the link? Not sure why you would keep it there 😕

Reply:
Fair, removed – I mean, it nevertheless is a concrete example of an email you can't copy-paste, but I can imagine someone booking via that link when they shouldn't, which is best avoided